### PR TITLE
Added full RP2040 Support

### PR DESCRIPTION
**Pull request by Intubun**
Like on the MCP_DAC Lib I added the 2nd Bus into the Library. Validaten with the Raspberry Pi Pico and the ADS1115.

### CODE REVIEW

File: ADS1X15.h [hunk 1]
```diff
@@ -39,16 +39,28 @@ class ADS1X15
...
   bool     isConnected();
 
   //       GAIN
-  // 0  =  �6.144V  default
-  // 1  =  �4.096V
-  // 2  =  �2.048V
-  // 4  =  �1.024V
-  // 8  =  �0.512V
-  // 16 =  �0.256V
+  // 0  =  �6.144V  default
```
Comment (RobTillaart) [inline ADS1X15.h:58]: looks like you lost the ±  sign here.
maybe just remove it.

**Comment by RobTillaart** [comment]
Overal it looks good, most important change I propose is to add the WIRE selection in the begin() function.

**Comment by RobTillaart** [comment]
@Intubun 
Thanks for testing, it looks perfect!👍
If build succeeds I will merge your changes. 

It might be an idea to check again on the MCP_DAC library. Think the same pattern can be used, simplifying the API.

**Comment by Intubun** [comment]
Yeah I think so too... Its much easier, if all Libs of a person are standardised. Im currently on the MCP_DAC Lib, The changes are working so far, but I need to change a whole lot, if I want to use the RP2040, since the "SPIClass" of the Pico is called "SPIClassRP2040" which is defenetly not great...

**Comment by RobTillaart** [comment]
Moved your post to a new thread under MCP_DAC to keep discussions under the right repo.

**Pull request merged**